bin/liquidator/src/rebalance.rs

```diff
@@ -69,9 +69,19 @@ impl Rebalancer {
             "checking for rebalance"
         );
 
-        self.rebalance_perps().await?;
-        self.rebalance_tokens().await?;
+        let rebalance_perps_res = self.rebalance_perps().await;
+        let rebalance_tokens_res = self.rebalance_tokens().await;
 
+        if rebalance_perps_res.is_err() && rebalance_tokens_res.is_err() {
+            anyhow::bail!(
+                "Failed to rebalance perps ({}) and tokens ({})",
+                rebalance_perps_res.unwrap_err(),
+                rebalance_tokens_res.unwrap_err()
+            )
+        }
+
+        rebalance_perps_res.expect("rebalancing perps failed");
+        rebalance_tokens_res.expect("rebalancing tokens failed");
```

Comment:
this means it'll panic if only the token or only the perp rebalancing fails, I'd prefer to use `_res?` here